protocol/messenger_mailserver.go

```diff
@@ -21,11 +21,15 @@ import (
 
 // tolerance is how many seconds of potentially out-of-order messages we want to fetch
 var tolerance uint32 = 60
+
 var mailserverRequestTimeout = 10 * time.Second
 var oneMonthInSeconds uint32 = 31 * 24 * 60 * 60
 var mailserverMaxTries uint = 2
 var mailserverMaxFailedRequests uint = 2
 
+// maxTopicsPerRequest sets the batch size to limit the number of topics per store query
+var maxTopicsPerRequest int = 20
```

Comment:
We were thinking of limiting it to 10, which is a lot. And recommend providing 2/3 as much (for the current SQLite implementation).

More or less, the reasoning is:
 
> A "broader" query, with many content topics (strings), implies many more comparisons during the query execution. This translates into slower queries.

Reply:
Just noticed the this PR's title states 10 topics, but you set the limit to 20 😜